### PR TITLE
Non latin chars in url

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: react/features/welcome/components/WelcomePage.native.js [hunk 1]
```diff
@@ -179,16 +182,23 @@ class WelcomePage extends AbstractWelcomePage {
...
+            if (!_roomNameValid) {
+                hintIdToShow = 'welcomepage.roomNameAllowedChars';
+            }
 
             return (
                 <Animated.View style = { this._getHintBoxStyle() }>
                     <View style = { styles.hintTextContainer } >
                         <Text style = { styles.hintText }>
-                            { t('welcomepage.roomnameHint') }
+                            { t(hintIdToShow) }
```
Comment (saghul) [inline react/features/welcome/components/WelcomePage.native.js]: does this look good if we use a ternary operator?

File: react/features/welcome/components/WelcomePage.web.js [hunk 2]
```diff
@@ -9,7 +9,11 @@ import { CalendarList } from '../../calendar-sync';
 import { RecentList } from '../../recent-list';
 import { SettingsButton, SETTINGS_TABS } from '../../settings';
 
-import { AbstractWelcomePage, _mapStateToProps } from './AbstractWelcomePage';
+import {
+    AbstractWelcomePage,
+    ROOM_NAME_VALIDATE_PATTERN_STR,
```
Comment (saghul) [inline react/features/welcome/components/WelcomePage.web.js]: style: all caps goes first

File: lang/main.json [hunk 3]
```diff
@@ -711,7 +711,7 @@
         "connectCalendarButton": "Connect your calendar",
         "connectCalendarText": "Connect your calendar to view all your meetings in {{app}}. Plus, add {{provider}} meetings to your calendar and start them with one click.",
         "enterRoomTitle": "Start a new meeting",
-        "onlyAsciiAllowed": "Meeting name should only contain latin characters and numbers.",
+        "roomNameAllowedChars": "Meeting name should not contain any of these characters: ?, &, :, ', \", %.",
```
Comment (saghul) [inline lang/main.json]: How about # ?

File: react/features/invite/components/info-dialog/web/InfoDialogButton.js [hunk 4]
```diff
@@ -260,7 +260,7 @@ function _mapStateToProps(state) {
             currentLiveStreamingSession
                 && currentLiveStreamingSession.liveStreamViewURL,
         _participantCount: getParticipantCount(state),
-        _toolboxVisible: state['features/toolbox'].visible
+        _toolboxVisible: true
```
Comment (virtuacoplenny) [inline react/features/invite/components/info-dialog/web/InfoDialogButton.js]: debug?

File: react/features/invite/components/info-dialog/web/InfoDialog.js [hunk 5]
```diff
@@ -388,6 +403,28 @@ class InfoDialog extends Component<Props, State> {
...
+
+    /**
+     * Callback invoked to copy the contents of {@code this._copyUrlElement} to the clipboard.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onCopyInviteUrl() {
+        try {
+            if (!this._copyElement) {
```
Comment (virtuacoplenny) [inline react/features/invite/components/info-dialog/web/InfoDialog.js]: Wrong variable being checked.

File: react/features/welcome/components/WelcomePage.web.js (hunk 2 above)
Comment (damencho) [reply react/features/welcome/components/WelcomePage.web.js]: Is this written somewhere where one can take a look?

File: react/features/invite/components/info-dialog/web/InfoDialogButton.js (hunk 4 above)
Comment (damencho) [reply react/features/invite/components/info-dialog/web/InfoDialogButton.js]: Ups.

File: lang/main.json (hunk 3 above)
Comment (damencho) [reply lang/main.json]: I've added that.

**Comment by damencho** [comment]
I think all comments are addressed, the mobile part of this was reverted as it is not needed and will break a feature where people can paste a full URL on the welcome page to pass parameters, and this is how mobile tests currently work.

**Pull request merged**